homework/hw03/task2.py

```diff
@@ -0,0 +1,12 @@
+# 2. Дан список повторяющихся элементов.
+# Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+my_list = [1, 2, 2, 3, 3, 3, 4, 4, 4, 4]
+
+
+def get_duplicates(input_list):
+    return list(set([x for x in input_list if input_list.count(x) > 1]))
```

Comment:
Используйте множество, чтобы уменьшить количество итераций, при этом не нужно переводить во множество, чтобы потом снова перевести в список.
```suggestion
    return [x for x in set(input_list) if input_list.count(x) > 1]
```